packages/commonwealth/client/scripts/views/components/proposals/proposal_extensions.tsx

```diff
@@ -31,8 +31,8 @@ export const ProposalExtensions = (props: ProposalExtensionsProps) => {
   } = props;
 
   React.useEffect(() => {
-    setDemocracyVoteAmount(0);
-    setCosmosDepositAmount(0);
+    if (setDemocracyVoteAmount) setDemocracyVoteAmount(0);
+    if (setCosmosDepositAmount) setCosmosDepositAmount(0);
```

Comment:
what is this doing?

Reply:
This is preventing these functions from running when they don't exist. This errors out when the chain does not have one of these functions otherwise